api/net/Program.cs

```diff
@@ -154,6 +154,7 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen(options =>
 {
+    options.ResolveConflictingActions(apiDescriptions => apiDescriptions.First());
```

Comment:
I'm curious what this does?

Reply:
sorry just saw your comment now -- i was getting an error: `Actions require unique method/path combination for Swagger 2.0` when running api swagger, this solves that error